common/processes/platform/tiny_process_library/tiny_process_library.cpp

```diff
@@ -17,30 +17,163 @@
 #include <string>
 #include <string_view>
 
+#include <common/events_io.hpp>
+#include <common/io.hpp>
+#include <common/log.hpp>
+
 using namespace std;
 
 namespace mender::common::processes {
 
+const chrono::seconds MAX_TERMINATION_TIME(10);
```

Comment:
Q: Since this is in essence making for the uniform behaviour of our process setup (It is part of the default construction). Should it not be a part of the header, like generate_line_data?

Reply:
Yes, but I wasn't planning to make this configurable, and in fact it could be a constant. The only reason I didn't make it a constant, but included it in construction instead, is so that the test can change its value and cut down on the testing time. If I didn't do this I would have doubled our current test run time!